AGGREGATE.md

```diff
@@ -108,14 +103,18 @@ which generates aggregation keys.
 [
 // Each dict independently adds pieces to multiple source keys.
 {
-  "key_piece": "10",// Conversion type purchase = 2
-  "key_offset": 9,
+  // Conversion type purchase = 2 at a 9 bit offset, i.e. 2 << 9.
+  // 9 bit offset is needed because there are 511 possible campaigns, which
+  // will take up 9 bits in the resulting key.
+  "key_piece": "0x400",
   // Apply this suffix to:
   "source_keys": ["campaignCounts"]
 },
 {
-  "key_piece": "10101",// Purchase category shirts = 21
-  "key_offset": 7,
+  // Purchase category shirts = 21 at a 7 bit offset, i.e. 21 << 7.
+  // 7 bit offset needed because there are 100 regions for the geo key, which
```

Comment:
"7 bit offset is needed"
"~100" to be consistent with comment in the source part.

Reply:
Done